jenkins/bootstrap.py

```diff
@@ -866,6 +867,7 @@ def bootstrap(args):
         else:
             version = ''
         setup_magic_environment(job)
+        setup_creds = True
```

Comment:
Done

Technically this was intentional -- if we blow up without setting up creds it will be inside the setup creds function... but I guess that will then result in weirder error messages than just retrying setup creds and blowing up there.